Signal/src/ViewControllers/Registration/RegistrationViewController.m

```diff
@@ -580,8 +581,10 @@ - (void)setDebugValue:(NSString *)value forKey:(NSString *)key
     OWSCAssert(value.length > 0);
 
     NSError *error;
-    [SAMKeychain setAccessibilityType:kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly];
```

Comment:
One small side benefit: we DRY up usage of `kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly`.